### PR TITLE
Docs: Fix Azure ad generic OAuth code markdown formatting

### PR DESCRIPTION
**Pull request by yesoreyeram**
For Azure AD oauth authentication,  `signout_redirect_url` needs to be specified for the user to be properly signed out.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by yesoreyeram** [comment]
This PR covers couple of fixes 

* Documented `signout_redirect_url ` in Azure AD config. Related issue https://github.com/grafana/grafana/issues/14160 ( @marefr mentioned this is not documented )
* Fix incomplete code snippets ( issue shown below screenshot )

image

File: docs/sources/auth/generic-oauth.md [hunk 1]
```diff
@@ -183,6 +183,7 @@ allowed_organizations =
     scopes = openid email name
     auth_url = https://login.microsoftonline.com/<directory id>/oauth2/authorize
     token_url = https://login.microsoftonline.com/<directory id>/oauth2/token
+    signout_redirect_url = https://login.microsoftonline.com/<directory id>/oauth2/logout?post_logout_redirect_uri=https://<grafana domain>
```
Comment (marefr) [inline docs/sources/auth/generic-oauth.md]: `signout_redirect_url` is not supported per OAuth provider, only on a global level in the `[auth]` section and is already documented [here](https://grafana.com/docs/grafana/latest/auth/overview/#url-redirect-after-signing-out).

**Review by marefr — CHANGES_REQUESTED** [review]
See comment. Only change left is then code markdown formatting fix.

**Review by marefr — APPROVED** [review]
LGTM

**Comment by marefr** [comment]
Thank you for contributing to Grafana!

**Pull request merged**